scripts/exact-example.jl

```diff
@@ -0,0 +1,229 @@
+# # Example 4: Exact diagonalisation
+
+# When working with smaller systems, or when multiple eigenvalues of a system are required,
+# it's better to work with an exact diagonalization method. There are a few ways to go about
+# this, each with its pros and cons. The purpose of this tutorial is to show off the methods
+# as well as provide a few tips regarding them.
+
+# A runnable script for this example is located
+# [here](https://github.com/joachimbrand/Rimu.jl/blob/develop/scripts/exact-example.jl).
+# Run it with `julia exact-example.jl`.
+
+# We start by loading the required modules.
+
+using Rimu
+using LinearAlgebra # eigen, eigvals, and eigvecs
+using Arpack        # eigs
+using KrylovKit     # eigsolve
+using Plots
+using LaTeXStrings
+gr() # hide
+nothing # hide
+
+# ## Preliminaries
+
+# In this example, we will look at a small example, a three-fermion Hamiltonian in one
+# dimension, formulated in momentum space. To start, we define some convenience functions.
+
+# In Rimu, momentum Fock states are stored such that the zero momentum state has a particle
+# in the middle of the address.
```

Comment:
I think you mean that a particle in the middle site of the lattice has zero momentum, not that the whole state has zero momentum. E.g. |1,0,1> also has zero (total) momentum.